### PR TITLE
Implement motion_change_recommendation creation in motion forward

### LINKED ISSUE #2519: Enhance ```motion.create_forwarded``` part 2
With the PR https://github.com/OpenSlides/openslides-backend/pull/2399 and https://github.com/OpenSlides/openslides-client/pull/3635
**Currently:**
If a motion (with change recommendations) is forwarded as original motion version then the recommendations become a part of the original motion

**Wanted:**
If a motion with change recommendations is forwarded as the original motion then the motion should have these change recommendations as change recommendations. 
I suppose that we need an additional flag to show that something has/is a change recommendation

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Closes #2519 

TODO:
- [x] Test with less attributes filled
- [x] Test with no change recommendations in db, but flag set
- [x] Test with change recos in db but no flag set
- [x] Test with amendments

### CODE REVIEW

**Review by bastianjoel — CHANGES_REQUESTED** [review]
Please update documentation

**Comment by luisa-beerboom** [comment]
> Please update documentation

Done

**Review by bastianjoel — APPROVED** [review]
Docs look good now

File: tests/system/action/motion/test_create_forwarded.py [hunk 1]
```diff
@@ -1316,3 +1316,225 @@ def test_name_generation(self) -> None:
...
+                "motion_id": created_id,
+                "meeting_id": 2,
+                "rejected": True,
+                "internal": True,
+                "type": "replacement",
+                "other_description": "Iamachangerecommendation",
+            },
+        )
+        assert reco["creation_time"] > 0
+        reco = self.assert_model_exists(
```
Comment (hjanott) [inline tests/system/action/motion/test_create_forwarded.py:1530]: `reco` isn't used after this line.

File: tests/system/action/motion/test_create_forwarded.py [hunk 2]
```diff
@@ -1316,3 +1316,225 @@ def test_name_generation(self) -> None:
...
```
Comment (hjanott) [inline tests/system/action/motion/test_create_forwarded.py:1380]: `reco` isn't used after this line.

**Review by hjanott — APPROVED** [review]

**Pull request merged**